stories/chartView.stories.tsx

```diff
@@ -100,7 +100,8 @@ class ChartStoryView extends React.Component {
 }
 
 export default {
-    title: "ChartView"
+    title: "ChartView",
+    component: ChartView
```

Comment:
What's this new component property?

Reply:
This is used to auto-generate documentation for the props: https://github.com/storybookjs/storybook/blob/next/addons/docs/react/README.md#props-tables

Looks like this:
![image](https://user-images.githubusercontent.com/2641501/89741403-55a1c680-da91-11ea-93f7-e5286140129f.png)